supertokens_python/recipe/usermetadata/__init__.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+#
+# This software is licensed under the Apache License, Version 2.0 (the
+# "License") as published by the Apache Software Foundation.
+#
+# You may not use this file except in compliance with the License. You may
+# obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Callable, Union
+
+from . import utils
+from .recipe import UserMetadataRecipe
+
+if TYPE_CHECKING:
+    from supertokens_python.supertokens import AppInfo
+
+    from ...recipe_module import RecipeModule
+
```

Comment:
You also need to expose the recipe functions in asyncio and syncio folders. See other recipes. I am talking about [these functions](https://github.com/supertokens/supertokens-node/pull/277/files#diff-6698c75ec26cd4e5c2594a2b892636768b9ad7eef3d2538096eca86cd5065609R22-R42) (from the node SDK)